custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py

```diff
@@ -376,15 +485,14 @@ def get_person_case_properties(domain, column_mapping, row):
     if properties["art_initiation_date"]:
         properties["art_initiated"] = "yes"
 
-    phone_number = column_mapping.get_value("phone_number", row),
+    phone_number = column_mapping.get_value("phone_number", row)
     if phone_number:
         properties['contact_phone_number'] = clean_phone_number(phone_number, 12)
         properties['phone_number'] = clean_phone_number(phone_number, 10)
         properties['language_code'] = "hin"
```

Comment:
i think it would be good to always set language code right? so that even if they don't have a phone number now they can add it later and we're ready to go